src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_BOUND = 100;
+
+    public Ball getRandomBall() {
+        Random randomVariables = new Random();
+        ColorSupplier colorVariables = new ColorSupplier();
```

Comment:
```suggestion
    private Random random = new Random();
    private ColorSupplier colorSupplier = new ColorSupplier();

    public Ball getRandomBall() {
```

Reply:
if you do this you won't need to create each time new instances of these classes when getRandomBall will be called, the same you should do with Random in ColorSupplier class